src/R/Components/Impl/PackageManager/Implementation/RPackageManager.cs

```diff
@@ -59,24 +59,6 @@ internal class RPackageManager : IRPackageManager {
             return await GetPackagesAsync(async (eval) => await eval.AvailablePackages());
```

Comment:
Should we perhaps prefetch data when package loads? Or is list population is now fast enough?